### PR TITLE
Add oot-u and oot-e to CI

### PR DESCRIPTION
**Pull request by Yanis42**
also this adds the badges to the readme

### CODE REVIEW

**Comment by Yanis42** [comment]
looks like it worked first try

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -42,8 +42,10 @@ jobs:
         PROGRESS_SLUG: oot-vc
         PROGRESS_API_KEY: ${{ secrets.PROGRESS_API_KEY }}
       run: |
-        python tools/upload_progress.py -b https://progress.decomp.club/ \
-          -p $PROGRESS_SLUG -v oot-j build/oot-j/progress.json
+        for version in {oot-j,oot-u,oot-e}; do
+          python tools/upload_progress.py -b https://progress.decomp.club/ \
```
Comment (cadmic) [inline .github/workflows/build.yml:46]: IIRC the version needs to be manually added for the upload to succeed? note that this part only runs on main so it's not tested in the PR build

Comment (Yanis42) [reply .github/workflows/build.yml:46]: yeah the version list needs to be updated there, I don't really like it but I guess we don't have the choice

Comment (Yanis42) [reply .github/workflows/build.yml:46]: wait do you mean on the distant website/database or do you mean in the workflow file? I was thinking of the workflow earlier

Comment (cadmic) [reply .github/workflows/build.yml:46]: yeah I meant on the website: https://github.com/zeldaret/oot-vc/pull/42#discussion_r1744215168

Comment (Yanis42) [reply .github/workflows/build.yml:46]: so from what I understand from the discord discussion (see https://discord.com/channels/688807550715560050/770656592504029194/1295186318413926503) it's already ready on the website

**Review by cadmic — APPROVED** [review]

**Pull request merged**